src/search/search.cpp

```diff
@@ -158,9 +159,13 @@ int32 main(int32 argc, char** argv)
         {
             logFile = argv[i + 1];
         }
+        if (strcmp(argv[i], "--append-date") == 0)
```

Comment:
can I get a newline between these `if` conditionals separating them from the preceding brace (in all 3 files)? thanks

Reply:
new lines have been added in the 3 files